package.json

```diff
@@ -628,7 +628,7 @@
         "test:setup":
             "yarn run build:test:integration && mocha -t 120000 --recursive lib_test/test/setup",
         "test:integration":
-            "yarn run build:test:integration && mocha -t 120000 lib_test/test/CiTests.js --bail",
+            "yarn run build:test:integration && mocha -t 120000 lib_test/test/CiTests.js --bail --runInBand --no-cache",
```

Comment:
Just for my info what does this change `runInBand`

Reply:
Yeah, apparently I got my wires crossed.  Looks like it has no effect.  It should ensure that all tests run in the same process.  But that looks to be a jest config option, not mocha.  From [the jest docs](https://jestjs.io/docs/en/cli.html#runinband)
> Run all tests serially in the current process, rather than creating a worker pool of child processes that run tests. This can be useful for debugging.

Reply:
I'm pushing a commit to remove those pointless flags.  Fingers crossed the tests still pass!